client/components/plans/plan-price/index.jsx

```diff
@@ -19,7 +19,7 @@ class PlanPrice extends React.Component {
 		let rawPrice, formattedPrice, months;
 
 		if ( plan ) {
-			// the properties of a plan object from sites-list is snake_case
+			// the properties of a plan object from sites list is snake_case
```

Comment:
Do we still have snake_case properties?

Reply:
@gwwar I don't know at all what the story of snake_case properties is. Is it something legacy that we can safely remove?

Reply:
From the comment, it sounded like sites-list was not normalizing properties like what we were doing in the redux store. So theoretically we can remove this entire comment.

Reply:
So I did a little research and learned that:
- site plans (`state.sites.plans`, the `getSitePlan` selector) have camelCase properties, because the API responses are transformed (search for `createSitePlanObject` function)
- global plans (`state.plans`, the `getPlanBySlug` selector) have snake_case properties, because the API responses are not transformed when receiving them

I updated the comment accordingly.

This whole component (`components/plans/plan-price`) will be removed soon anyway, because it's used only by `PremiumPopover`, used in legacy (non-Kracken) domains search. @jsnmoon has been landing relevant PRs in last few days.